truckersmp_cli/args.py

```diff
@@ -83,12 +83,20 @@ def check_args_errors():
 Need to download (-u) Proton?""".format(Args.protondir))
 
     # checks for updating
-    if Args.update and not Args.account:
-        if VDF_IS_AVAILABLE:
-            Args.account = get_current_steam_user()
+    if Args.update:
         if not Args.account:
-            logging.info("Unable to find logged in steam user automatically.")
-            sys.exit("Need the steam account name (-n name) to update.")
+            if VDF_IS_AVAILABLE:
+                Args.account = get_current_steam_user()
+            if not Args.account:
+                logging.info("Unable to find logged in steam user automatically.")
+                sys.exit("Need the steam account name (-n name) to update.")
+
+        # check whether Proton is present
+        # when "--skip-update-proton" is specified
+        if (Args.skip_update_proton
+                and not os.access(os.path.join(Args.protondir, "proton"), os.R_OK)):
+            sys.exit(
+                "'--skip-update-proton' was specified but Proton is not found.")
```

Comment:
I can't quite follow you here but maybe I'm scrambling things around:

You're checking for proton while skipping proton during an update.
I don't think we need proton to update so it doesn't matter if proton is or isn't available at update time.

Instead we need a check if we're starting and updating and skipping protons update in the process.